### PR TITLE
User's favorites

### PR DESCRIPTION
**Pull request by june29**
make user's favorite effects page

### CODE REVIEW

File: app/views/users/favorites.html.haml [hunk 1]
```diff
@@ -8,23 +8,8 @@
       %h3
         %i.icon-star
         お気に入りエフェクト
```
Comment (hmsk) [inline app/views/users/favorites.html.haml:10]: any other naming?

Comment (june29) [inline app/views/users/favorites.html.haml:10]: "お気に入りのエフェクト"...

File: app/views/users/_contributions.html.haml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.user-contributions
+  %h3
+    %i.icon-flag
+    関わったイベント
+  .stats
+    = "参加%d / 運営%d / 発表%d / 視聴%d / ドラ娘%d" % [:attendee, :staff, :talker, :watcher, :dorar].map { |role| user.contributions.where(role => true).count}
+  - if contributions.empty?
+    - if user == current_user
```
Comment (hmsk) [inline app/views/users/_contributions.html.haml:8]: shall we change this condition to instance variable? ex:) `@is_own_page?`

File: app/views/users/favorites.html.haml [hunk 3]
```diff
@@ -8,23 +8,8 @@
...
-      - if @user == current_user
-        .alert.alert-info
-          = link_to "開催イベントの一覧", events_path
-          からあなたが関わったイベントを見つけて、どんな関わり方をしたのか登録しよう！
-      - else
-        %p
-          まだ、登録されていません。
-    - else
-      = render :partial => "contributions/line", :as => :contribution, :collection => @contributions
+    = render :partial => "users/contributions", :as => :user, :object => @user, :locals => { :contributions => @contributions }
```
Comment (hmsk) [inline app/views/users/favorites.html.haml:15]: :thumbsup:

File: app/views/users/show.html.haml [hunk 4]
```diff
@@ -13,16 +13,4 @@
...
-        - if @user == current_user
-          .alert.alert-info
-            = link_to "開催イベントの一覧", events_path
-            からあなたが関わったイベントを見つけて、どんな関わり方をしたのか登録しよう！
-        - else
-          %p
-            まだ、登録されていません。
-      - else
-        = render :partial => "contributions/line", :as => :contribution, :collection => @contributions
+      = render :partial => "users/contributions", :as => :user, :object => @user, :locals => { :contributions => @contributions }
```
Comment (hmsk) [inline app/views/users/show.html.haml:16]: :thumbsup:

File: app/views/users/favorites.html.haml (hunk 1 above)
Comment (hmsk) [inline app/views/users/favorites.html.haml:10]: "好きなエフェクト"、"☆を付けたエフェクト"...

**Comment by hmsk** [comment]
I've put some comments but LGTM about this commit.

**Comment by hmsk** [comment]
if you don't have any other commit, please merge.

File: app/views/users/_contributions.html.haml (hunk 2 above)
Comment (june29) [inline app/views/users/_contributions.html.haml:8]: it's better!

**Pull request merged**